tasks/task_database.c

```diff
@@ -313,11 +313,10 @@ static int database_info_list_iterate_found_match(
 
    if(!playlist_entry_exists(playlist, entry_path_str, db_crc))
    {
-      // For the playlist item name, use the game's
-      // description if it's less then 60 characters
-      // long. Otherwise, just use the name.
+      // For ScummVM, use the game description, for every
+      // other system, use the DAT's game name.
       char* name;
-      if (strlen(db_info_entry->description) <= 60)
+      if (strcmp(db_playlist_base_str, "ScummVM.lpl") == 0)
```

Comment:
> use another field to store long descriptions.

Now it only uses the description field for ScummVM. Having the DAT contain the name means you can run clrmamepro on the DAT on your game datafiles, and it structures the files in the folders for ScummVM correectly.